system-tests/e2e-suite/src/test/scala/com/hotels/styx/plugins/DoubleSubscribingPluginSpec.scala

```diff
@@ -65,9 +63,7 @@ class DoubleSubscribingPluginSpec extends FunSpec
 
 
   describe("Styx as a plugin container") {
-
-    // TODO: Mikko: Styx 2.0 API: Test fails. Look into it.
-    ignore("Tolerates plugins that break the content observable chain") {
+    it("Tolerates plugins that break the content observable chain") {
       val testClient = aggregatingTestClient("localhost", styxServer.httpPort)
 
       withTestClient(testClient) {
```

Comment:
I think this test should assert that a PluginError has been logged. It could also assert that relevant metrics have been incremented.

Otherwise this test suggests there is nothing unusual about breaking a content observable chain.